core/src/main/java/psiprobe/tools/AccessorFactory.java

```diff
@@ -43,7 +43,13 @@ public static Accessor getInstance() {
   private static Accessor getReflective() {
     try {
       return new ReflectiveAccessor();
-    } catch (Exception ex) {
+    } catch (ClassNotFoundException e) {
```

Comment:
https://developers.google.com/java-dev-tools/codepro/doc/?hl=en

Eclipse will complain because this hasn't been updated for 4+ yet.  The 3.7
version still works in Luna.

On Sun, Feb 7, 2016, 11:30 PM Jeremy Landis <notifications@github.com>
wrote:

> In core/src/main/java/psiprobe/tools/AccessorFactory.java
> <https://github.com/psi-probe/psi-probe/pull/655#discussion_r52132908>:
>
> > @@ -43,7 +43,13 @@ public static Accessor getInstance() {
> >    private static Accessor getReflective() {
> >      try {
> >        return new ReflectiveAccessor();
> > -    } catch (Exception ex) {
> > +    } catch (ClassNotFoundException e) {
>
> What is CodePro? First I've heard of that.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/psi-probe/psi-probe/pull/655/files#r52132908>.
>